airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java

```diff
@@ -2093,7 +2093,10 @@ public void updateUser(HttpServletRequest request, HttpServletResponse response)
         if (u == null) {
             error(request, response, ErrorCode.NOT_FOUND, "No such user: " + username);
             return;
-        } else if (org.airsonic.player.domain.User.USERNAME_ADMIN.equals(username)) {
+        } else if (user.getUsername().equals(username)) {
+            error(request, response, ErrorCode.NOT_AUTHORIZED, "Not allowed to change own user");
```

Comment:
What about "You're not allowed to update the properties of your own user" instead?

Reply:
Since these messages are in the REST API, I kept them very terse, as no one should ever see them except developers.